WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/ShippingLabelPackateItemViewModel.swift

```diff
@@ -9,6 +9,19 @@ final class ShippingLabelPackageItemViewModel: ObservableObject {
     ///
     let selectedPackageID: String
 
+    @Published var totalWeight: String = ""
+
+    /// The items rows observed by the main view `ShippingLabelPackageDetails`
```

Comment:
In this case, these items rows are only observed by the `ShippingLabelPackageItem` view, right?

Reply:
Good catch - thanks!